src/renderer/components/icon/icon.tsx

```diff
@@ -59,10 +60,7 @@ export class Icon extends React.PureComponent<IconProps> {
 
         // fallthrough
       case "Enter": {
-        // eslint-disable-next-line react/no-find-dom-node
-        const icon = findDOMNode(this) as HTMLElement;
-
-        setTimeout(() => icon.click());
+        setImmediate(() => this.ref?.click());
```

Comment:
We should omit `setImmediate`. It works without it and `setImmediate` doesn't have enough browser support https://developer.mozilla.org/en-US/docs/Web/API/Window/setImmediate